microsoft/testsuites/dpdk/dpdksuite.py

```diff
@@ -168,6 +169,41 @@ def _check_rx_or_tx_pps(
                 "than 2^20 (~1m) PPS after sriov disable."
             ).is_less_than(2 ** 20)
 
+    @TestCaseMetadata(
+        description="""
+            verify vpp is able to detect azure network interfaces
+            - run fd.io vpp install scripts
+            - install vpp from their repositories
+            - start vpp service
+            - check that azure interfaces are detected by vpp
+        """,
+        priority=2,
+        requirement=simple_requirement(
+            min_nic_count=2,
+            network_interface=Sriov(),
+        ),
+    )
+    def verify_dpdk_vpp(
+        self, node: Node, log: Logger, variables: Dict[str, Any]
+    ) -> None:
+
+        vpp = DpdkVpp(node)
+        vpp.install()
+
+        net = Nics(node)
```

Comment:
use `node.nics`. It can cache the information, reduce calls.

Reply:
Ah nice, does that mean I don't need initialize call then?

Reply:
The initialization is handled in nodes.